### ISSUE TITLE
Display timespent on minutes + seconds only

### ISSUE THREAD

**Issue by mcruzel** [catalyst/moodle-block_dedication#83]
Hello,

Some Dean of Studies use Excel extracts generated by this plugin. Problem is that the new timespent format (hh:mm:ss) isn't convinient for excel treatments (average, filters, sorting...).

Could you implement the possibility to add a colum "timespent in seconds" and "timespent in minutes", at least by a single calculation on Excel export ?

Thank you in advance

Best regards

MC

**Comment by danmarsden** [comment]
Hi there - yes that sounds like something that could be useful to add - if you have internal development capabilities, pull requests are always welcome, otherwise feel free to reach out privately if you would like to fund development on the plugin!

thanks!

**Comment by azimshaikh95** [comment]
I have raised a "pull request" ([#95](https://github.com/catalyst/moodle-block_dedication/pull/95))
It gives the timespent in as single unit
For Example: 	0009 hour 35 min 08 sec = 9.586 Hours

-It will be easy to export & sort / split in excel.
-Also works with Aggregation (Average, Sum, etc) in Moodle Custom Report Builder.

**Comment by danmarsden** [comment]
Thanks - that's not quite what I was thinking... I was thinking we'd just add a new column to the report builder entity that doesn't call format_duration and allows the raw duration value to be included. eg this code; (not change the behaviour of the existing column at all.)

eg take a look here:
https://github.com/catalyst/moodle-block_dedication/blob/MOODLE_400_STABLE/classes/local/entities/dedication.php#L92-L103

basically use that to create a new column - maybe "timespentraw" that allows the column to be displayed in a format that suits you better - then you can add that column to the report when you want to use it.

**Comment by azimshaikh95** [comment]
Yes, we can. But I'm no much expert with PHP coding. So I just observed the code and made some tweaks to get output in a single unit.

Actually first I tried to add another column but I didn't able to add it. So I have to skip it.

if you can create a new column, it would be great. Thanks

**Comment by danmarsden** [comment]
thanks - feel free to submit a new PR in future if you manage to work this out! :-)
